styles/Odoo/WordList.yml

```diff
@@ -33,25 +29,23 @@ swap:
   authZ: authorization
   autoupdate: automatically update
   check box|check-box: checkbox
-  CLI: command-line tool
-  curated roles: predefined roles
+  drop down|dropdown: drop-down menu
   data are: data is
   disabled?: turn off|off
+  drag and drop: drag-and-drop
+  e-learning|Elearning: eLearning
   fewer data: less data
+  gantt: Gantt
   file name: filename
   firewalls: firewall rules
   functionality: capability|feature
   grayed-out: unavailable
-  HTTPs: HTTPS
+  http|https: HTTP|HTTPS
+  Kanban: kanban
```

Comment:
I'm worried about this line because while this rule catches
"Take a look at this in kanban view", there's a false positive when a sentence begins with the word
"Kanban dashboards are a useful tool!"

Approving this update for now, but i want to bring a potential edge case to light

Reply:
We can add a grouping and character class to only apply this rule if kanban is used in the middle of words rather than at the beginning of a sentence.

```
'(?:[^a-z]Kanban)': kanban

✅                                                                       ❌                 ✅
Kanban dashboards are a useful tool! It is not Kanban this. Kanban is a good term.
```